web/src/message_edit.js

```diff
@@ -960,6 +967,7 @@ export function save_message_row_edit($row) {
         rows.get_message_recipient_header($row).attr("data-stream-id"),
         10,
     );
+    assert(message_lists.current !== undefined);
```

Comment:
Again, probably more readable to just have this before the variable definitions, rather than breaking them up with it.

Reply:
Done.